pkg/k8s/k8s.go

```diff
@@ -133,8 +151,9 @@ func NewService(n *v1alpha1.Nginx) *corev1.Service {
 					Port:       int32(80),
 				},
 			},
-			Selector: LabelsForNginx(n.Name),
-			Type:     nginxService(n),
+			Selector:       LabelsForNginx(n.Name),
+			LoadBalancerIP: lbIP,
```

Comment:
Interesting... This way we can tie an external (outside of cluster) IP address to the Service, right?!

Reply:
Yep, it depends on the cloud-provider supporting it though:
```
$ k explain svc.spec.loadBalancerIP
KIND:     Service
VERSION:  v1

FIELD:    loadBalancerIP <string>

DESCRIPTION:
     Only applies to Service Type: LoadBalancer LoadBalancer will get created
     with the IP specified in this field. This feature depends on whether the
     underlying cloud-provider supports specifying the loadBalancerIP when a
     load balancer is created. This field will be ignored if the cloud-provider
     does not support the feature.
```

Reply:
Thanks.